src/main/utils/PlaybackAPI.js

```diff
@@ -35,6 +42,10 @@ class PlaybackAPI {
         album: null,
         albumArt: null,
       },
+      rating: {
+        liked: false,
```

Comment:
Why are we translating a number in the 0-5 range into 2 booleans.  If users have the 5 star experiment turned on this could be any value (1, 2, 3, 4, 5)